devlib/native/win_native.cpp

```diff
@@ -168,6 +169,53 @@ namespace winutil {
     }
 
 
+    static auto findBusNumber(DEVINST const& handle, QMap<QString, int> & cachedDeviceBuses) {
+        static auto rootHubsBuses = initializeRootBuses();
+        qDebug() << "Buses" << rootHubsBuses;
+
+
+        auto currentDevInst = handle;
+        DEVINST parentDevInst;
+        WCHAR instanceID [MAX_DEVICE_ID_LEN];
+
+        auto visitedDevices = QSet<QString>{};
```

Comment:
I'd rather name it `traversedDevices` otherwise it sounds weird as if devices are some sights or your friends

Reply:
Done